frontend/static/js/services/loginService.js

```diff
@@ -1,25 +1,19 @@
 import backendURL from "../constants.js";
+import RequestService from "./requestService.js";
 
-async function postLogin({ username, password }) {
-	const request = fetch(backendURL.loginURL, {
-			method: "POST",
-			headers: {
-			"Content-Type": "application/json",
-			},
-			body: JSON.stringify({
+class LoginService extends RequestService {
+	constructor() {
+		super();
+	}
+
+	async postRequest({username, password}) {
+		return super.postRequest(
+			`${backendURL.loginURL}`,
+			JSON.stringify({
 				username: username,
 				password: password,
-			}),
-		})
-	return request.then((response) => {
-		if (response.ok) {
-			const newData = response.json();
-			return newData;
-		} else {
-			throw new Error(`${response.status}`);
-		}
-	})
+		}));
+	}
```

Comment:
here is where we should add those other api requests and throw notimplemented as they are not meant to be called yet right?